process/compas.py

```diff
@@ -0,0 +1,112 @@
+import json
+
+import numpy as np
+import pandas as pd
+
+
+class Params:

```

Comment:
I think all this Params thing is a lot of overhead that we don't really need here. As far as I can see we could remove all that and just hard-code the positive/negative outcome? WDYT?